src/test/java/org/opentripplanner/routing/edgetype/TimetableTest.java

```diff
@@ -286,10 +306,12 @@ public void testUpdate() {
         stopTimeUpdateBuilder.setScheduleRelationship(
                 StopTimeUpdate.ScheduleRelationship.SCHEDULED);
         stopTimeEventBuilder = stopTimeUpdateBuilder.getDepartureBuilder();
-        stopTimeEventBuilder.setDelay(-1);
+        stopTimeEventBuilder.setDelay(1);
         tripUpdate = tripUpdateBuilder.build();
         updatedTripTimes = timetable.createUpdatedTripTimes(tripUpdate, timeZone, serviceDate); 
         assertNotNull(updatedTripTimes);
+        // Ensure that times past the updated one is still valid.
+        assertEquals(0, updatedTripTimes.getDepartureDelay(0));
         timetable.setTripTimes(trip_1_1_index, updatedTripTimes);
```

Comment:
Agreed, it should probably read "stops upstream of the updated one are still valid, but have no delay"